assets/js/guides.js

```diff
@@ -113,4 +113,12 @@ $(document).ready(function () {
   $('.post-detail img').on('click', function() {
     window.open(this.src, '_blank')
   })
-});
\ No newline at end of file
+
+  //Add ga tracking on clicking of the subscribe cta within the guides
+  $('.admonitionblock a').on('click', function() {
+    if($(this).text().toLowerCase().includes('subscriber')){
+      ga('send', 'event', location.pathname, 'cta-subscribe');
```

Comment:
Hm, this seems a bit too magical and brittle... 

I believe our goal should be:

1. For the "you must be a subscriber" CTA, we pop-up the modal. It looks like this code only sends tracking?
1. We want to do show this modal and capture these clicks in a way that isn't too brittle. 

Let's instead add [AsciiDoc roles](https://asciidoctor.org/docs/user-manual/#role) to those links in the deployment guides. Those roles translate to CSS class names that we can target more explicitly.

Reply:
I am triggering the modal [here](https://github.com/gruntwork-io/gruntwork-io.github.io/blob/master/assets/js/main.js#L49) and yes the code sends only tracking. I've modified the code to retrieve the class name instead